pages/_app.js

```diff
@@ -14,13 +19,16 @@ function App({ Component, pageProps }) {
       <GlobalStyles />
       <ThemeProvider theme={theme}>
         <StyledLayout>
-          <StyledHeader />
+          <StyledHeader>
+            <Header />
+          </StyledHeader>
```

Comment:
```suggestion
          <Header />
```

move the max-width css to the header component and turn the header component into a header. Wrapping the header component like this is unnecessary